tools/font-icon/src/index.js

```diff
@@ -81,22 +84,26 @@ async function loadSvgFont(file) {
   return await xml2js.parseStringPromise(svgFile);
 }
 
-async function build(){
-  let config;
+async function build(_config, _isProd = isProd, _dry = false){
+  let config = _config;
   try {
-    config = require(path.resolve(configFile));
+    if (!config) {
+      config = require(path.resolve(configFile));
+    }
```

Comment:
Can we only read config from _config? Use Object.assign?

Reply:
We still want to support the config file way,
So it the user doesn't need to write js, if they prefer cli